src/DssSpell.sol

```diff
@@ -19,22 +19,29 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-import { DssInstance, MCD } from "dss-test/MCD.sol";
-import { D3MInit, D3MCommonConfig, D3MAavePoolConfig, D3MAaveBufferPlanConfig } from "src/dependencies/dss-direct-deposit/D3MInit.sol";
-import { D3MInstance } from "src/dependencies/dss-direct-deposit/D3MInstance.sol";
-
-interface PoolConfiguratorLike {
-    function setReserveFreeze(address asset, bool freeze) external;
-    function setReserveInterestRateStrategyAddress(address asset, address newRateStrategyAddress) external;
-    function setReserveFactor(address asset, uint256 newReserveFactor) external;
+interface StarknetLike {
+    function setCeiling(uint256 _ceiling) external;
+}
+
+interface GemLike {
+    function transfer(address, uint256) external returns (bool);
+    function allowance(address, address) external view returns (uint256);
+    function approve(address, uint256) external returns (bool);
+}
+
+interface VestLike {
+    function file(bytes32, uint256) external;
+    function restrict(uint256) external;
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function yank(uint256) external;
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/98e98eae03662eeab0dd2092ccc7edafb2dd75d3/governance/votes/Executive%20vote%20-%20April%2028%2C%202023.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/3e28783bb8e5dc0ad2ea2597cf5ae825b6ee7ee8/governance/votes/Executive%20vote%20-%20May%2010%2C%202023.md' -q -O - 2>/dev/null)"
```

Comment:
Apparently this got updated recently:

```suggestion
    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/8d9266124fcf45a8065ecc50b43ef1c8b6f099ce/governance/votes/Executive%20vote%20-%20May%2010%2C%202023.md' -q -O - 2>/dev/null)"
```

The content hash wasn't changed though.